CustomerData/CartTagging.php

```diff
@@ -122,7 +130,9 @@ public function getSectionData()
         }
 
         if ($data["itemCount"] > 0) {
-            $this->updateNostoId();
+            $store = $this->scopeHelper->getStore();
+            $data['restore_cart_url'] = $this->nostoRestoreCartUrlBuilder
```

Comment:
I would add `$data['restore_cart_url'] = null` also to here https://github.com/Nosto/nosto-magento2/blob/develop/CustomerData/CartTagging.php#L98 to have the same data structure regardless if the cart has items or not